pkg/mcproducer/scripts/replace-kernel-module.sh

```diff
@@ -59,6 +73,8 @@ if [ -n "$(podman images -q $full_kernel_module_image 2> /dev/null)" ]; then
         echo "failed to insert OOT kernel module $kernel_module"
     fi
     podman wait $worker_pod_id
+    echo "removing kmm-pod"
+    podman pod rm $worker_pod_name
```

Comment:
Won't the pod be automatically removed if all its containers are using the `--rm` flag?

Reply:
the pod remains with status of `Exited`, even though the containers were removed